apps/website/src/pages/docs/slider.mdx

```diff
@@ -9,19 +9,50 @@ group: inputs
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='input-slider' />
-
 <LiveExample src='Slider.main.tsx' truncate={false}>
   <AllExamples.SliderMainExample client:load />
 </LiveExample>
 
-Sliders allow the user to graphically select a specific input value, or a range, within defined limits. Optionally, a tooltip appears above the slider to indicate the current value and changes as the slider is repositioned. The double sliders should be used when the desired input is a range (for example, 33 through 55). In this case, the tooltip indicates the current range and the area between the two sliders is highlighted.
+Sliders allow the user to graphically select a specific input value, or a range, within defined limits. Optionally, a [tooltip](tooltip) appears above the slider handle to indicate the current value and changes as the slider handle is repositioned. The sliders min, max, and step values can be changed as needed. To help visually enforce incremental steps, you may optionally choose to display ticks underneath the slider. The slider can be oriented horizontally or vertically.
+
+Changes made with sliders should be immediate, allowing a user to make slider adjustments until finding their preference. This means a setting controlled by a slider should not require to be validated by clicking a button for the changes to be applied.
+
+## Variants
+
+### With labels & steps
+
+<LiveExample src='Slider.labels.tsx'>
+  <AllExamples.SliderLabelsExample client:load />
+</LiveExample>
+
+You can display tick labels underneath the slider. You can also have the value jump by a certain amount of steps - in this example it jumps by 10. The min and max labels can be text or icons.
+
+### Range
+
+<LiveExample src='Slider.range.tsx'>
```

Comment:
I thought we do as you wrote in the issue:
> Each section should be ordered title, paragraph, demo, additional paragraphs if needed.

Is it ok to have title, demo, paragraph? We should choose one.

Reply:
Apologies, I forgot to go back and update the issue. #961 has been updated to reflect this new order.

Reply:
I have not been diligent in checking this structure in other PRs either. We can easily change the order once all said is done.